### PR TITLE
Enable feedback in apps

### PR DESCRIPTION
**Pull request by dapenguin**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "o-footer": "^6.0.0",
     "o-header": "^7.5.6",
     "o-tracking": "^1.3.8",
-    "logo-images": "^1.5.2"
+    "logo-images": "^1.5.2",
+    "n-feedback": "^0.0.6"
```
Comment (kavanagh) [inline bower.json]: It would be better to start at `1.0.0-beta.1`

File: browser/js/app-initializer.js [hunk 2]
```diff
@@ -27,6 +28,7 @@ export const presets = {
 		ads: true,
 		syndication: true,
 		roe: true,
+		feedback: true,
```
Comment (kavanagh) [inline browser/js/app-initializer.js]: Is it safe to make it on by default yet? Yes we should probably have it default to on when we're happy with the UI, but are we ok with doing that right now?

File: browser/js/app-initializer.js [hunk 3]
```diff
@@ -146,11 +148,15 @@ export class AppInitializer {
...
+				if (this.enabledFeatures.feedback) {
+					feedback.init();
+				}
+
 				if (this.enabledFeatures.roe) {
 					roe.init(flags);
 				}
 
-				if (this.enabledFeatures.evenMoreJanky) {
+				if (this.enabledFeatures.evenMoreJanky && flags.get('qualtrics')) {
```
Comment (kavanagh) [inline browser/js/app-initializer.js]: Is this the correct place for the flag?  I would have though it should be on line 151.

Comment (dapenguin) [reply browser/js/app-initializer.js]: Poop! Good spot

**Pull request merged**